setuptools/config.py

```diff
@@ -240,6 +240,13 @@ def _parse_bool(cls, value):
         value = value.lower()
         return value in ('1', 'true', 'yes')
 
+    @classmethod
+    def _exclude_files_parser(cls, value):
+        exclude_directive = 'file:'
+        if value.startswith(exclude_directive):
+            raise ValueError('Only strings are accepted, Files are not accepted')
```

Comment:
I think this should be `files` instead of `Files`, but I'm a bit worried that this will be a confusing error message if we can't tell them the field that caused the problem.